### PR TITLE
CIV-15290 CoSC - set CoSCApplicationStatus to Active on successful payment of application fee

### PR DESCRIPTION
**Pull request by hmcts-version1-ignacio**
### I WANT TO set the CoSCApplicationStatus to Active on successful payment for (FEE0459)
SO THAT I can process the application accordingly ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json [hunk 1]
```diff
@@ -86,5 +86,13 @@
     "FieldTypeParameter": "TranslatedDocument",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "coSCApplicationStatus",
+    "Label": "CoSC Application Status",
+    "FieldType": "YesOrNo",
```
Comment (sherlynkhaw) [inline ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json:94]: should be fixed list - active, processed, inactive

File: ga-ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec-CUI-nonprod.json [hunk 2]
```diff
@@ -1768,5 +1768,17 @@
         "CRUD": "CRUD"
       }
     ]
+  },
```
[the remark this answers is not in the document]
Comment (LeonardoPalmeiro) [reply ga-ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec-CUI-nonprod.json]: I removed.. Thanks

File: ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json (hunk 1 above)
Comment (LeonardoPalmeiro) [reply ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json]: I removed because this fields should be on main case.

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by kalachandrasekar1 — DISMISSED** [review]

File: ga-ccd-definition/CaseEvent/User/UserEventsGAspec-JO-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "INITIATE_COSC_APPLICATION_AFTER_PAYMENT",
+    "Name": "Cosc Applic After Payment",
```
Comment (ShwetaTandel-hmcts) [inline ga-ccd-definition/CaseEvent/User/UserEventsGAspec-JO-nonprod.json:5]: Cosc Application After Payment - would be better instead of applic

Comment (sherlynkhaw) [reply ga-ccd-definition/CaseEvent/User/UserEventsGAspec-JO-nonprod.json:5]: i think it might hit the character limit?

Comment (sherlynkhaw) [reply ga-ccd-definition/CaseEvent/User/UserEventsGAspec-JO-nonprod.json:5]: actually that's bang on 30 characters! will push the changes

**Review by ShwetaTandel-hmcts — DISMISSED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Pull request merged**